cypress/e2e/title-level-request/user-can-create-page-Item-level-request.cy.js

```diff
@@ -0,0 +1,166 @@
+    requestTypes: ['Page'],
+    name: `recall_${getRandomPostfix()}`,
+    id: uuid(),
+  };
+  before('Preconditions', () => {
+    cy.getAdminToken()
+      .then(() => {
+        ServicePoints.createViaApi(testData.userServicePoint);
+        testData.defaultLocation = Location.getDefaultLocation(testData.userServicePoint.id);
+        Location.createViaApi(testData.defaultLocation);
+        cy.getInstanceTypes({ limit: 1 }).then((instanceTypes) => {
+          testData.instanceTypeId = instanceTypes[0].id;
+        });
+        cy.getHoldingTypes({ limit: 1 }).then((holdingTypes) => {
+          testData.holdingTypeId = holdingTypes[0].id;
+        });
+        cy.createLoanType({
+          name: `type_${getRandomPostfix()}`,
+        }).then((loanType) => {
+          testData.loanTypeId = loanType.id;
+        });
+        cy.getMaterialTypes({ limit: 1 }).then((materialTypes) => {
+          testData.materialTypeId = materialTypes.id;
+        });
+      })
+      .then(() => {
+        InventoryInstances.createFolioInstanceViaApi({
+          instance: {
+            instanceTypeId: testData.instanceTypeId,
+            title: instanceData.title,
+          },
+          holdings: [
+            {
+              holdingsTypeId: testData.holdingTypeId,
+              permanentLocationId: testData.defaultLocation.id,
+            },
+          ],
+          items: [
+            {
+              barcode: testData.itemBarcode,
+              status: { name: 'Available' },
+              permanentLoanType: { id: testData.loanTypeId },
+              materialType: { id: testData.materialTypeId },
+            },
+          ],
+        });
+      });
+    PatronGroups.createViaApi(patronGroup.name).then((patronGroupResponse) => {
+      patronGroup.id = patronGroupResponse;
+    });
+    RequestPolicy.createViaApi(requestPolicyBody);
+    CirculationRules.getViaApi().then((circulationRule) => {
+      originalCirculationRules = circulationRule.rulesAsText;
+      const ruleProps = CirculationRules.getRuleProps(circulationRule.rulesAsText);
+      ruleProps.r = requestPolicyBody.id;
+      CirculationRules.addRuleViaApi(originalCirculationRules, ruleProps, 't ', testData.loanTypeId);
+    });
+
+    cy.createTempUser(
+      [
+        permissions.uiUsersfeefinesCRUD.gui,
+        permissions.uiUsersfeefinesView.gui,
+        permissions.loansRenew.gui,
+        permissions.requestsAll.gui,
+      ],
+      patronGroup.name
+    ).then((userProperties) => {
+      userData = userProperties;
+      UserEdit.addServicePointViaApi(
+        testData.userServicePoint.id,
+        userData.userId,
+        testData.userServicePoint.id
+      );
+      cy.loginAsAdmin({
+        path: SettingsMenu.circulationTitleLevelRequestsPath,
+        waiter: TitleLevelRequests.waitLoading,
+      }).then(() => {
+        TitleLevelRequests.changeTitleLevelRequestsStatus('allow');
+      });
+      cy.login(userData.username, userData.password);
```

Comment:
You can use here like example cy.login(user.username, user.password, { path:TopMenu.ordersPath, waiter: Orders.waitLoading }); 
and remove first cy.visit from it

Reply:
done